packages/material-ui-codemod/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@material-ui/codemod",
-  "version": "5.0.0-alpha.30",
```

Comment:
I was wondering whether this is fine when making the release. Should we increase the version by one, or always bump all packages when doing a release so they would be the same version.

Reply:
We currently aim to use a **single** version. Otherwise, as a developer, you have to check if each package is truly on the latest version.